swawozny/form-app
# Feature form validation

Validation:
1. Check if the continent is Europe, and the last name has a length of less than 2 characters.
2. Check if the first name exists.
3. Check if the birthday is not after today.
4. Check if the user is not older than 60 years.
